tests/unittests/test_http_functions.py

```diff
@@ -10,8 +10,8 @@
 
 import pytest
 
-from azure_functions_worker import testutils
-from azure_functions_worker.testutils import WebHostTestCase
+from tests.utils import testutils
+from tests.utils.testutils import WebHostTestCase
```

Comment:
double import - why not refer to `testutils.WebHostTestCase`? Like in other locations?

Reply:
Fixed.